spec/tapioca/compilers/dsl/active_record_columns_spec.rb

```diff
@@ -414,6 +414,9 @@ def author_previous_change; end
               sig { returns(T::Boolean) }
               def author_previously_changed?; end
 
+              sig { returns(T.nilable(::String)) }
```

Comment:
Is this because we bumped AR?

Reply:
Yes, I think it is related to this PR upstream: rails/rails#36836